Source/WebKit/GPUProcess/media/RemoteAudioSessionProxy.cpp

```diff
@@ -120,12 +120,22 @@ void RemoteAudioSessionProxy::configurationChanged()
 
 void RemoteAudioSessionProxy::beginInterruption()
 {
-    connection().send(Messages::RemoteAudioSession::BeginInterruption(), { });
+    connection().send(Messages::RemoteAudioSession::BeginInterruptionRemote(), { });
```

Comment:
Remote is redundant, everything here is always remote.
Otherwise you end up with method have Remote twice.

Reply:
The naming is not redundant, see the ChangeLog:

```
Rename the IPC method to avoid conflicting the method on the `AudioSession` base
class that drives interruptions. This change is necessary to avoid sending a
spurious IPC to the GPU process when the interrupted state is changed in the Web
process.
```